services/match/src/events/consumer.ts

```diff
@@ -43,6 +43,8 @@ async function consumeMatchUpdated(msg: MatchUpdatedEvent) {
 
 async function consumeCollabCreated(msg: CollabCreatedEvent) {
     const { requestId1, requestId2, collabId } = msg;
+    // Remember remove this console.log after debugging is done.
+    console.log(msg);
```

Comment:
```suggestion
    // Remember remove this console.log after debugging is done.
    console.log(msg);
```

Remember to remove this 🤣